headers/augment_headers.py

```diff
@@ -289,6 +293,13 @@ def add_header_content(
         action="store_true",
         help="add alias declarations",
     )
+    parser.add_argument(
+        "-ad",
+        "--mark_aliases_deprecated",
+        action="store_true",
+        help="mark aliases as deprecated (if --aliases is set)",
+        default=True
```

Comment:
I'd rather we not set a default (i.e., default to None/False). The syntax used isn't actually part of standard C, it's a GCC extension. Also, I think if you have a `store_true` flag that defaults to true, there's no way to actually turn it off?